hypothesis/lms
# Integrate OneDrive file picker

I created the `OneDrivePickerClient` class that uses the
`loadOneDriveAPI` to load the OneDrive JS bundle asynchronously.

`OneDrivePcikerClient` has a single method, `showPicker` to launch the
OneDrive picker. It is a callback based method.

To test this PR:

* Login into http://localhost:8001/admin
* Find  the following consumer key: `Hypothesisb3be0b33d0b5e4b1cf3aaf04e0e1819a`
* Click on the `Microsoft OneDrive enabled` checkbox
* Click Save

Then, you should be able to see the `Select PDF from OneDrive` option when setting an assignment in LMS:

![Screenshot 2021-09-22 at 14 52 03](https://user-images.githubusercontent.com/8555781/134346879-de48dc3c-dc47-4e2b-92ce-a4beebedf732.png)
